modules/interconnect-config/added/add_connectors.py

```diff
@@ -0,0 +1,97 @@
+# FILE: add_connectors.py
+# DESC: Adds connectors to running router pods
+#################################################
+import time
+import json
+import httplib
+import ssl
+import os
+import sys
+
+TOKEN_FILE  = "/var/run/secrets/kubernetes.io/serviceaccount/token"
+CONFIG_FILE = sys.argv[1]
+
+IP=0 # Field to reference pod IP address
+TS=1 # Field to reference pod timestamp
+
+def retrieve_token():
+  with open(TOKEN_FILE, 'r') as f:
+    return f.read().replace('\n', '')
+
+def api_request(host, port, path, token):
+  ''' Makes REST request to API server and returns response'''
+  context = ssl._create_unverified_context()
+  conn = httplib.HTTPSConnection(host, port, timeout=5, context=context)
+  conn._context.check_hostname = False
+  conn._context.verify_mode = ssl.CERT_NONE
+  #conn.set_debuglevel(1)
+
+  headers = {}
+  headers['Authorization'] = "Bearer %s" % token
+
+  req = conn.request("GET", path , headers=headers)
+  res = conn.getresponse()
+
+  return res.read()
+
+def extract_ips(response, label):
+  ''' Given an API response, return list of pod ips ordered by pod start time'''
+  pod_list = []
+  js = json.loads(response)
+
+  for pod in js['items']:
```

Comment:
When running on Openshift 3.9 getting the following error:
...
ConfigMap volume mounted, copying over configuration files ...
  | Traceback (most recent call last):
  | File "/opt/interconnect/bin/add_connectors.py", line 86, in <module>
  | ip_list  = extract_ips(response, name)
  | File "/opt/interconnect/bin/add_connectors.py", line 42, in extract_ips
  | for pod in js['items']:
  | KeyError: 'items'
  | 2018-04-24 01:11:30.658600 +0000 SERVER (info) Container Name: router-amq-1-b6x74
...

You can see the deployment at https://ce-os2-rhel-master.usersys.redhat.com:8443/console/project/router/overview
login as: vbusch

And a similar error on openshift 3.7 https://ce-os-rhel-master.usersys.redhat.com:8443/console/project/interconnect/overview